substrate/frame/support/procedural/src/construct_runtime/parse.rs

```diff
@@ -171,7 +161,7 @@ pub enum WhereKind {
 pub struct WhereDefinition {
 	pub kind_span: Span,
 	pub kind: WhereKind,
-	pub value: syn::TypePath,
+	pub _value: syn::TypePath,
```

Comment:
Why do you keep all these field if they are not used?

Reply:
Makes sense. Removed them.